src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -84,6 +84,8 @@ def load_arguments(self, _):
                        help='Runtime version of used language')
             c.argument('jvm_options', type=str,
                        help="A string containing jvm options, use '=' instead of ' ' for this argument to avoid bash parse error, eg: --jvm-options='-Xms1024m -Xmx2048m'")
+            c.argument('net_core_main_entry_path', type=str,
```

Comment:
@yucwan Could you please work with @metacpp about the command parameter interface for steeltoe?

Reply:
the name seems too long to me. Suggest "main_entry_path" or "main_entry". It's enough to have .net in the discription.